app/src/main/java/com/google/android/gms/samples/wallet/PaymentsUtil.java

```diff
@@ -268,6 +254,18 @@ private static JSONObject getMerchantInfo() throws JSONException {
                     "allowedPaymentMethods", new JSONArray().put(PaymentsUtil.getCardPaymentMethod()));
             paymentDataRequest.put("transactionInfo", PaymentsUtil.getTransactionInfo(price));
             paymentDataRequest.put("merchantInfo", PaymentsUtil.getMerchantInfo());
+
+            // An optional shipping address requirement is a top-level property of the PaymentDataRequest
+            // JSON object. Omitting all together means all countries are supported.
```

Comment:
remove "omitting all together" sentence. it is incorrect, as country restrictions is in a different property